docs/make.jl

```diff
@@ -3,6 +3,10 @@ using Documenter, DataFramesMeta
 makedocs(
 	modules = [DataFramesMeta],
 	sitename = "DataFramesMeta Documentation",
+	format = Documenter.HTML(
+		canonical = "https://juliadata.github.io/DataFramesMeta.jl/stable/",
+		assets = ["assets/favicon.ico"]
```

Comment:
I think you need to add the icon to docs/src/assets/ (you can copy it from DataFrames). Or just drop this line.